byoeb-v1/byoeb/byoeb/apis/chat.py

```diff
@@ -65,3 +70,52 @@ async def delete_collection(
             content=f"Error: {e}",
             status_code=500
         )
+
+def chat_mcps_router(mcp):
+    @mcp.tool
+    async def asha_chat(message: str) -> str:
+        """
+        Ask any health-related query and get a response.
+        """
+        phone_number = mcp_get_phone_number()
+        user_id = get_user_ids_from_phone_number_ids([phone_number])[0]
+        users = await dependency_setup.user_db_service.get_users([user_id])
+        if len(users) == 0:
+            return "Before I can answer your question, you must register yourself as an ASHA user. Shall I start with the registration?"
+
+        user = users[0]
+        ctx = ByoebMessageContext(
+            channel_type="whatsapp",
+            message_category="whatsapp",
+            user=user,
+            message_context=MessageContext(
+                message_id="chat-mcps-router-for-" + user_id,
+                message_type=MessageTypes.REGULAR_TEXT.value,
+                message_source_text=message,
+                message_english_text=message,
+                media_info=None,
+                additional_info=dict(query_type="asha_work_related")
+            ),
+            reply_context=ReplyContext(
+                reply_id="reply-id-unknown",
+                reply_type="acknowledgement",
+                reply_source_text=message,
+                reply_english_text=message,
+                media_info=None,
+                message_category="notification",
+                additional_info=None
+            ),
+            cross_conversation_id=None,
+            cross_conversation_context=None,
+            incoming_timestamp=None,
+            outgoing_timestamp=None
+        )
+        responses = await dependency_setup.byoeb_user_generate_response.handle_message_generate_workflow([ctx])
+        for resp in responses:
+            if resp.message_category == "bot_to_asha_response":
```

Comment:
Instead of hardcoding this in byoeb\models\message_category.py there is enum present for it we can use that.
```suggestion
from byoeb.models.message_category import MessageCategory
if resp.message_category == MessageCategory.BOT_TO_USER_RESPONSE.value:
```